src/scripts/modules/workspace/popovers/new/modals/new-media.coffee

```diff
@@ -26,6 +26,7 @@ define (require) ->
       else
         content = new Page
           title: options.title
+          content: '<p>Enter some content here. Format content, and drag and drop elements from the toolbar.</p>'
```

Comment:
The stand alone page does not show the default text when the content is set in toJSON method.